trilium_alchemy/core/note/note.py

```diff
@@ -878,7 +877,7 @@ def __ixor__(
 
         return self
 
-    def __getitem__(self, key: str) -> str | Note:
+    def __getitem__(self, key: str) -> str | Note | None:

```

Comment:
This method can't actually return `None` since line 888 would raise a `KeyError` if there weren't any attributes with that name. I think the `if len(attrs)` check can be simply omitted, resulting in no code paths where we return `None`.